src/sonic-config-engine/minigraph.py

```diff
@@ -1526,9 +1548,10 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         for key in voq_inband_intfs:
            results['VOQ_INBAND_INTERFACE'][key] = voq_inband_intfs[key]
 
-
     if resource_type is not None:
         results['DEVICE_METADATA']['localhost']['resource_type'] = resource_type
+        if 'Appliance' in resource_type:
```

Comment:
>Appliance [](http://example.com/codeflow?start=12&length=9)

Should it be `HaAppliance` or ending with `Appliance` ?

Reply:
it can be any combination, but expect the keyword 'Appliance'. Its already deployed with 201911